### DISCUSSION TITLE
nixos flake install steps

### DISCUSSION THREAD

**Discussion by dezren39** [juspay/omnix#228] [Q&A] [answered]
hey so how do i install this into my nixos flake?

**Comment by srid** [comment, marked as answer]
See https://github.com/juspay/nix-dev-home/commit/f6b9a4d50ddf114c97c1ca419738866900a694cf

Basically, add `omnix.url = "github:juspay/omnix";` as flake input, and access the package as `inputs.omnix.packages.${pkgs.system}.default`.

**Reply by dezren39** [reply]
thank you!
